cmd/kuberhealthy/main.go

```diff
@@ -34,6 +36,7 @@ import (
 var kubeConfigFile = filepath.Join(os.Getenv("HOME"), ".kube", "config")
 var listenAddress = ":8080"
 var podCheckNamespaces = "kube-system"
+var dnsEndpoints = []string{}
```

Comment:
you don't need to make a change here, but you could instead write this as `var dnsEndpoints []string`